pages/about.html

```diff
@@ -0,0 +1,103 @@
+                <p class="ADuviri col-lg-6 col-6 col-sm-6 offset-lg-3 offset-3 offset-sm-3">Perhaps the duviri paradox is closer than you think</p>
+                <ul class="row col-lg-10 col-10 col-sm-10 offset-lg-1 offset-1 offset-sm-1">
+                    <li class="AImage1 col-lg-5 col-10 col-sm-10">2022<p class="SubText">Plains of Eidolon</p></li>
+                    <li class="AImage2 col-lg-5 col-10 col-sm-10">2023<p class="SubText">Fortuna</p></li>
+                </ul>
+            </div>
+        </div>
+      </section>
+      <section id="AFooterTop">
+        <div class="nav_top row col-lg-12 col-12 col-sm-12">
+            <div class="col-lg-1 col-3 col-sm-3 offset-lg-3 offset-1 offset-sm-1"><img class="col-lg-12 col-12 col-sm-12" src="../images/Warframelogo.jpg" alt="Warframelogo"></div>
+            <ul class="nav_top col-lg-6 col-6 col-sm-6">
+                <li class="ANavBottom col-lg-12 col-12 col-sm-12">NINJAS PLAY FREE</li>
+                <li class="ANavBottom col-lg-12 col-12 col-sm-12">Calculating intelligence profile of the grineer...ERROR: not a number...did the operator enjoy this witticism.</li>
+            </ul>
+        </div>
+      </section>
+      <section id="MobileAFooterTop">
+        <div class="nav_top col-lg-12 col-12 col-sm-12">
+            <div class="Platform col-lg-2 col-2 col-sm-2 offset-5">Platforms</div>
+            <div class="col-lg-2 offset-lg-5 col-2 col-sm-2 offset-sm-5 offset-5"><hr class="Line col-lg-2 offset-lg-5 col-2 col-sm-2 offset-sm-5 offset-5"></div>
+            <ul class="nav_top row col-lg-12 col-12 col-sm-12">
+                <li class="col-3 col-sm-3"><img class="Console col-12 col-sm-12" src="../images/PC.jpg" alt="PC"></li>
+                <li class="col-3 col-sm-3 offset-1 offset-sm-1"><img class="Console col-12 col-sm-12" src="../images/PS4.jpg" alt="PS4"></li>
+                <li class="col-3 col-sm-3 offset-1 offset-sm-1"><img class="Console col-12 col-sm-12" src="../images/PS5.jpg" alt="PS5"></li>
+                <li class="col-3 col-sm-3 offset-2 offset-sm-2"><img class="Console col-12 col-sm-12" src="../images/xboxone.jpg" alt="xboxone"></li>
+                <li class="col-3 col-sm-3 offset-1 offset-sm-1"><img class="Console col-12 col-sm-12" src="../images/seriesx.jpg" alt="seriesx"></li>
+            </ul>
+        </div>
+      </section>
+      <section id="AFooterBottom">
+        <div class="nav_top row col-lg-12 col-12 col-sm-12">
+            <div class="col-3 col-sm-3 offset-1"><img class="ABLogo col-12 col-sm-12" src="../images/Warframelogo.jpg" alt="Warframelogo"></div>
+            <ul class="nav_top col-7 col-sm-7 offset-1">
+                <li class="Ninjas col-lg-12 col-12 col-sm-12">NINJAS PLAY FREE</li>
+            </ul>
+        </div>
+      </section>
```

Comment:
- [ ] These two sections are supposed to be in the `footer` tag because it's a footer, please kindly fix that 🙃